### PR TITLE
chore: support go1.14

### PR DESCRIPTION
**Pull request by appleboy**
as title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -400,4 +401,4 @@ Become a bronze sponsor with a monthly donation of $10 and get your logo on our
 
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/WeChatPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/AliPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
-<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
\ No newline at end of file
+<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
```
Comment (panjf2000) [inline README.md:404]: This is a little bit weird, additional lines are the same as deleted lines? How come?

File: README_ZH.md [hunk 2]
```diff
@@ -401,4 +402,4 @@ pool.Reboot()
 
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/WeChatPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/AliPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
-<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
\ No newline at end of file
+<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
```
Comment (panjf2000) [inline README_ZH.md:405]: As above.

File: README.md (hunk 1 above)
Comment (appleboy) [reply README.md:404]: My editor (`VSCode`) will auto removes the additional `space` or `tab` at end of line after saving the file.

Comment (panjf2000) [reply README.md:404]: I see.

**Comment by panjf2000** [comment]
Thanks~

**Pull request merged**